gap/ReducibleMatrixGroups.gi

```diff
@@ -358,3 +358,104 @@ function(d, q, k)
 
     return MatrixGroupWithSize(F, generators, size);
 end);
+
+
+# Construction as in Proposition 4.3 of [HR05]
+BindGlobal("SpStabilizerOfIsotropicSubspace",
+function(d, q, k)
+    local field, gens, I, J, GLgen, Xi, Spgen, Yi;
+
+    if IsOddInt(d) then
+        ErrorNoReturn("<d> must even but ", d, " was given.");
```

Comment:
This line is not covered. Of course, it's an error, and you are careful to not trigger errors in your tests.

That said, in general I've come to find it useful to explicitly also add tests for functions with incorrect arguments, to verify that error handling is in place and working. As a bonus side effect, this also allows to get code coverage (closer) to 100%. It also helps to find things like formatting mistakes in error messages.

Note a requirement, thought, just a suggestion.